srcpkgs/x-unikey/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'x-unikey'
+pkgname=x-unikey
+version=1.0.4
+revision=1
+build_style=gnu-configure
+makedepends="libX11-devel"
+short_desc="Vietnamese input method for X"
+maintainer="Linh Nguyễn <linhvng.2212@gmail.com>"
+license="GPL-2.0-only"
```

Comment:
Most of them are `GPL-2.0-or-later`, some files are licensed under `MIT`.

Reply:
sorry, it's my first time adding a package and tracking down the license :sweat_smile: thought it was GPL2 because of the COPYING file... what license should i change it to?